src/main/java/com/metsci/laproc/plotting/ROCCurve.java

```diff
@@ -24,11 +28,55 @@ public ROCCurve(ClassifierDataSet in) {
      * @return The plottable data set representing this curve
      */
     public GraphableData compute() {
-        GraphableData d = new Curve("ROC");
-        d.addPoint(0.1, 0.4);
-        d.addPoint(0.3, 0.4);
-        d.addPoint(0.5, 0.5);
-        d.addPoint(0.7, 0.8);
+        GraphableData d = new Curve("ROC", NUMPOINTS);
```

Comment:
To try to keep the code readable, d, p, val, tpr, and fpr should probably be written out.